test/Spaces/spaces.jl

```diff
@@ -53,6 +53,44 @@ import ClimaCore.DataLayouts: IJFH
           Spaces.column(coord_data, 1, 1)[]
 end
 
+@testset "extruded (2d 1×3) finite difference space" begin
+
+    FT = Float32
+
+    vertdomain = Domains.IntervalDomain(
+        Geometry.ZPoint{FT}(0),
+        Geometry.ZPoint{FT}(10);
+        boundary_tags = (:bottom, :top),
+    )
+    vertmesh = Meshes.IntervalMesh(vertdomain, Meshes.Uniform(), nelems = 10)
+    vert_face_space = Spaces.FaceFiniteDifferenceSpace(vertmesh)
+    # Generate Horizontal Space
+    horzdomain = Domains.IntervalDomain(
+        Geometry.XPoint{FT}(0),
+        Geometry.XPoint{FT}(10);
+        periodic = true,
+    )
+    horzmesh = Meshes.IntervalMesh(horzdomain; nelems = 5)
+    horztopology = Topologies.IntervalTopology(horzmesh)
+    quad = Spaces.Quadratures.GLL{4}()
+    hspace = Spaces.SpectralElementSpace1D(horztopology, quad)
+    # Extrusion
+    f_space = Spaces.ExtrudedFiniteDifferenceSpace(hspace, vert_face_space)
+    c_space = Spaces.CenterExtrudedFiniteDifferenceSpace(f_space)
+    array = parent(Spaces.coordinates_data(c_space))
+    z = Fields.coordinate_field(c_space).z
+    @test size(array) == (10, 4, 17, 5) # 10V, 4I, 2F(x,z), 5H 
+    @test size(parent(z[colidx])) == (10, 1)
+
+    # Define test col index
+    colidx = ClimaCore.Fields.ColumnIndex{1}((4,), 5)
+    # Here valid `colidx` are `ClimaCore.Fields.ColumnIndex{1}((1:4,), 1:5)`
+    @test Fields.field_values(z[colidx]) isa ClimaCore.DataLayouts.VF
+    @test Spaces.column(z, 1, 1, 1) isa ClimaCore.Fields.Field
+    @test_throws BoundsError Spaces.column(z, 1, 2, 1)
```

Comment:
@simonbyrne This was the original error when using the 3 argument `column` method corresponding to the `VIJFH` layouts in the `Fields.bycolumn` block.